llvm/include/llvm/IR/IntrinsicInst.h

```diff
@@ -569,6 +569,9 @@ class VPIntrinsic : public IntrinsicInst {
   /// The llvm.vp.* intrinsics for this instruction Opcode
   static Intrinsic::ID getForOpcode(unsigned OC);
 
+  /// The llvm.vp.reduce.* intrinsics for this intrinsic.
```

Comment:
`intrinsic for this intrinsic`.

Also, doesn't this function now return more than just reduction intrinsics? It handles all intrinsic->vp.intrinsic mappings we have.

Should we unit test this function?

Reply:
Might want to document what happens when passed VP intrinsics, too

Reply:
165f0bcf2dc7bf3f3fe98dc18c3f81e622255a67  Fix the comment.
be39dca2226bff98a292c2542ff884a49c521d14  This commit defined the behavior when passed VP intrinsics. I  define the behavior to return the passed VP intrinsic ID directly. Please let me know if you feel this is unsuitable. 
86f96f1f8a4b4867351f0e5bfc7f7d97c1e889f7 Add round trip tests for `getForIntrinsic`.